product_docs/docs/sql_assess/1/using.mdx

```diff
@@ -0,0 +1,260 @@
+
+After generating the trace files using either the Oracle trace utility or the EDB Query Extractor, the user can run the `assessSQL.jar` file using the `runSQLAssess.sh` script to generate a report. The report helps the user to identify the compatibility of the application for porting the application to EDB Postgres Advanced Server. The tool also generates SQL output files. The queries in these output files can then be modified to be compatible with EDB Postgres Advanced Server syntax. 
+
+!!! Note 
+    The suggested query syntax for procedure calls in the generated SQL output files is compatible with executing those calls via JDBC and other languages. For example, a procedure call like:
+
+    ```
+     begin procin (:1 ); end;
+    ```
+
+    Is converted to the following call:
+
+    ```
+    {CALL procin (? ) }
+    ```
+
+### Command options
+
+```shell
+/usr/edb/sqlassess/bin/runSQLAssess.sh -help
+usage: assessSQL <options>
+ -appname <name>             Specify the name of the application
+ -help                       Print all available options
+ -includeUIDs <userIDs>      Specify comma separated list of User IDs for
+                             assessment consideration
+ -inputfilepath <path>       Specify the path containing the input file/s
+ -inputfiletype <filetype>   Specify either trace, exported or plain
+ -loglevel <level>           Specify logging level e.g. off, info, fine
+ -reportpath <path>          Specify the path where report will be saved
+ -sourcedb <version>         Specify source database e.g. oracle_19c
+ -targetdb <version>         Specify target database e.g. epas_15
+ -version                    Print the version information and exit
+
+
+/usr/edb/sqlassess/bin/runSQLAssess.sh -version
+EDB SQL Assess ver 1.0.0-preview
+
+/usr/edb/sqlassess/bin/runSQLAssess.sh -appname TeleApp -inputfilepath /some/valid/path/ -inputfiletype plain -reportpath /some/valid/path/ -sourcedb Oracle_19c -targetdb epas_15 -loglevel FINE
+
+EDB SQL Assess ver 1.0.0-preview
+Input file type: PLAIN
+Input file path: /home/edb/Projects/
+Report path: /home/edb/Projects/
+Application Name: TeleApp
+Source Database: ORACLE_19C
+Target Database: EPAS_15
+Logging Level: FINE
+Processing File: t0.sql...
+Generating Report....
+Connected to H2 ..
+**********************
+Generating report ....
+**********************
+Report generation complete.
+Another example providing only the mandatory command line options.
+
+./runSQLAssess.sh -inputfilepath /home/edb/ -inputfiletype trace -reportpath /home/edb/
+EDB SQL Assess ver 1.0.0-preview
+Input file type: TRACE
+Input file path: /home/edb/
+Report path: /home/edb/
+Application Name: TestApp
+Source Database: ORACLE_19C
+Target Database: EPAS_15
+Logging Level: INFO
+Processing File: orcl_ora_117637.trc...
+Generating Report....
+Connected to H2 ..
+**********************
+Generating report ....
+**********************
+Report generation complete.
+```
+
+### `includeUIDs` parameter
```

Comment:
I believe the include UIDs parameter is only relevant for trace and possibly exported files.  There should be some text indicating that it is not relevant for plain SQL files (and exported files if it is not).

@gabbasb can you confirm?

Reply:
@mlewandowski-edb The includeUIDs parameter is only relevant for trace files. It is not relevant for exported or plain SQL files.
I agree we should add an example in Command options section explaining the usage of includeUIDs as follows:

Example providing includeUIDs, which is relevant only in case of trace files.

./runSQLAssess.sh -includeUIDs 109,93 -inputfilepath /home/edb/ -inputfiletype trace -reportpath /home/edb/
EDB SQL Assess ver 1.0.0-preview
Input file type: TRACE
Input file path: /home/edb/
Report path: /home/edb/
Application Name: TestApp
Source Database: ORACLE_19C
Target Database: EPAS_15
Logging Level: INFO
Processing File: ORCL_ora_35652.trc...
Generating Report....
Connected to H2 ..
**********************
Generating report ....
**********************
Report generation complete.

Reply:
I added the following in the opening paragraph:

> The `includeUIDs` parameter doesn't filter queries from exported or plain SQL files. 

And, an Example subsection with the above example that @gabbasb supplied.